source/_components/sensor.fitbit.markdown

```diff
@@ -22,6 +22,7 @@ Enable the sensor by adding the following to your `configuration.yaml` file:
 sensor:
   - platform: fitbit
     clock_format: 12H
+    unit_system: metric
```

Comment:
Please remove this line, we are trying to keep the initial examples as minimal as possible. Therefore, optional configuration variables should be left out.

Reply:
Done